Dockerfile

```diff
@@ -3,9 +3,10 @@ FROM alpine:3.9
 LABEL maintainer="technology@werkspot.com"
  
 RUN apk add --no-cache varnish
+RUN chown nobody:nogroup -R /etc/varnish
```

Comment:
Why do we need to change ownership of this directory? In most cases, a volume should be mounted in this directory. The mounted directory can have any owner you'd like.

Reply:
In this microservice we built we have a Dockerfile extending `werkspot/varnish` which does a `COPY` instruction, hence needing the right permission when executing `varnishd` with `/etc/varnish/default.vcl`

Reply:
Just tested this version and it works like a char